packages/mermaid/src/diagrams/common/common.spec.ts

```diff
@@ -69,6 +69,5 @@ describe('generic parser', () => {
       'test <Array<Array<string[]>>>'
     );
     expect(parseGenericTypes('~test')).toEqual('~test');
-    expect(parseGenericTypes('~test Array~string~')).toEqual('~test Array<string>');
```

Comment:
`~test Array~string~` renders as `~test Array<string>` currently. 

Removing that is a breaking change in the behaviour of diagrams, which we cannot have. 

Added support back in https://github.com/mermaid-js/mermaid/pull/4805

Reply:
Actually, this PR has been open for a long time. No need to delay is further. 
I'll merge this and point #4805 to develop